packages/thumbprint-atomic/package.json

```diff
@@ -5,6 +5,7 @@
     "author": "Tom Genoni <tom@thumbtack.com>",
     "main": "dist/atomic.css",
     "style": "dist/atomic.css",
+    "type": "module",
```

Comment:
One thing that may be worth testing is whether this will cause issues with the clients that use Thumbprint Atomic. IIRC, this tells Node that the package is uses ES Module syntax. I've run into problems in the past with packages that do this (new versions of `lodash-es`, in particular) since we don't transpile `node_modules` by default.

It might be worth testing this out by adding this line to a `node_modules/@thumbtack/thumbprint-atomic/package.json` file in some project and then running the project to see if it builds.

Reply:
Yeah, the situation with Typescript / ts-node / ESM / native ES modules is a mess right now. I found all sorts of conflicting answers on GH/SO. I'll run your test and report back. I'm hoping it's ok since this is a CSS only package

Reply:
yep, added `"type": "module",` to `/Users/glavelle/Thumbtack/frontend/node_modules/@thumbtack/thumbprint-atomic/package.json` and Nextjs builds fine

Reply:
Ok cool, thanks for checking!